tuplex/codegen/include/IFailable.h

```diff
@@ -30,6 +30,8 @@ enum class CompileError {
     TYPE_ERROR_RETURN_ITERATOR,
     TYPE_ERROR_NEXT_CALL_DIFFERENT_DEFAULT_TYPE,
     TYPE_ERROR_MIXED_ASTNODETYPE_IN_FOR_LOOP_EXPRLIST, // exprlist contains a mix of tuple/list of identifiers and single identifier
+    TYPE_ERROR_TYPE_UNSTABLE_IN_LOOP,
+    COMPILE_ERROR_ALL_SAMPLES_PRODUCE_NORMALCASEVIOLATION,
```

Comment:
can you give @bgivertz  a ping that this error is new? He might need it for his project.

Reply:
Sure.